### PR TITLE
organize import sections with rustfmt `group_imports`

### PR DESCRIPTION
**Pull request by onur-ozkan**
ref https://rust-lang.zulipchat.com/#narrow/stream/326414-t-infra.2Fbootstrap/topic/development.20plans.20for.20bootstrap/near/390360397

### CODE REVIEW

File: src/debuginfo/mod.rs [hunk 1]
```diff
@@ -5,21 +5,19 @@ mod line_info;
...
-
+pub(crate) use emit::{DebugReloc, DebugRelocName};
 use gimli::write::{
     Address, AttributeValue, DwarfUnit, FileId, LineProgram, LineString, Range, RangeList,
     UnitEntryId,
 };
 use gimli::{Encoding, Format, LineEncoding, RunTimeEndian};
 use indexmap::IndexSet;
-
-pub(crate) use emit::{DebugReloc, DebugRelocName};
```
Comment (bjorn3) [inline src/debuginfo/mod.rs:20]: Please revert this change. This is an import of a crate-local item and as such should be separate from the other imports. Maybe adding a `self::` here and on the line below would convince rustfmt to keep them separate?

File: src/intrinsics/mod.rs [hunk 2]
```diff
@@ -19,16 +19,15 @@ mod llvm_x86;
 mod simd;
 
 pub(crate) use cpuid::codegen_cpuid_call;
+use cranelift_codegen::ir::AtomicRmwOp;
 pub(crate) use llvm::codegen_llvm_intrinsic_call;
```
Comment (bjorn3) [inline src/intrinsics/mod.rs:23]: Similarly here. `cpuid` and `llvm` are submodules.

File: src/debuginfo/mod.rs [hunk 3]
```diff
@@ -5,21 +5,19 @@ mod line_info;
...
-
 use gimli::write::{
     Address, AttributeValue, DwarfUnit, FileId, LineProgram, LineString, Range, RangeList,
     UnitEntryId,
 };
 use gimli::{Encoding, Format, LineEncoding, RunTimeEndian};
 use indexmap::IndexSet;
-
-pub(crate) use emit::{DebugReloc, DebugRelocName};
 pub(crate) use unwind::UnwindContext;
```
Comment (bjorn3) [inline src/debuginfo/mod.rs:16]:
```suggestion
pub(crate) use self::unwind::UnwindContext;
```

File: src/debuginfo/mod.rs [hunk 4]
```diff
@@ -5,21 +5,19 @@ mod line_info;
...
 
+pub(crate) use self::emit::{DebugReloc, DebugRelocName};
+use crate::prelude::*;
```
Comment (bjorn3) [inline src/debuginfo/mod.rs:19]:
```suggestion

pub(crate) use self::emit::{DebugReloc, DebugRelocName};
pub(crate) use self::unwind::UnwindContext;
use crate::prelude::*;
```

**Comment by onur-ozkan** [comment]
I start feeling scared. I thought rustfmt will handle this in a better way. How are we going to detect this on the upstream from 1333 files :joy:

**Comment by bjorn3** [comment]
(currently working on fixing CI. will merge after that.)

**Comment by bjorn3** [comment]
The CI issue should be fixed now, but still waiting on confirmation from CI.

**Comment by bjorn3** [comment]
Thanks!
